api/compute/solution_request.go

```diff
@@ -681,7 +683,11 @@ func (s *SolutionRequest) dispatchSolution(statusChan chan SolutionStatus, clien
 				break
 			}
 		}
-		produceSolutionRequest := createProduceSolutionRequest(produceDatasetURI, fittedSolutionID, outputKeys)
+		exposeType := []string{}
+		if s.useParquet {
+			exposeType = append(exposeType, "PARQUET_URI")
```

Comment:
Should add a constant to `ta3ta2.go` for this string and use that instead.  The other value types all have defs there.